cmd/pj-rehearse/main.go

```diff
@@ -87,9 +87,21 @@ an infrastructure issue.`
 
 pj-rehearse rehearsed jobs and at least one of them failed. This means that
 job would fail when executed against the current HEAD of the target branch.`
+
+	deleteConfigMapsFailureOutput = ` [ERROR] pj-rehearse: delete configmaps failure
+
+pj-rehearse could not delete the collection of the temporary configmaps that have
+been created for this execution. That means that they will still exist in the cluster
+and any administrator should be notified for manual integration.`
```

Comment:
Shouldn't we clean up at the start of the run or use a replacing call? "Contact an admin" doesn't scale

Reply:
This is our last line of defense!! If that fails, and the pj-rehearse tool will exit, how we will manage to delete those configmaps? Do you think that its a better idea to add some `ttl` label and update the ttl controller to clean up also configmaps?

Reply:
@stevekuznetsov @petr-muller any thoughts on that?

Reply:
Hash the input files to the maps, so it doesn't ever collide, and do an out-of-band cleanup IMO

Reply:
I am not afraid of a collision.  If the delete will fail for another reason that it's related to the cluster, the cm will stay there forever.

Reply:
addressed